lib/config/types.ts

```diff
@@ -95,6 +95,7 @@ export interface GlobalOnlyConfig {
   privateKeyPathOld?: string;
   redisUrl?: string;
   repositories?: RenovateRepository[];
+  endpoint?: string;
```

Comment:
I think it should be removed from here, as it's not used global-only (we will use it in datasource, for example).